tenant-isolation-config-service-api/src/main/proto/org/hypertrace/tenantisolation/config/service/v1/tenant_isolation_group_config.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package org.hypertrace.tenantisolation.config.service.v1;
+
+option java_multiple_files = true;
+
+message TenantIsolationGroupConfig{
+  string id = 1;
+  string group_name = 2;
+  repeated string members = 3;
```

Comment:
be specific - what does a member string represent? Guessing it should be `member_tenant_ids`?